modules/sops/default.nix

```diff
@@ -133,6 +146,12 @@ in {
       '';
     };
 
+    log = mkOption {
+      type = types.listOf (types.enum [ "gpgImport" "changes" ]);
+      default = [ "gpgImport" "changes" ];
```

Comment:
Maybe we should rename this to `keyImport`? I assume most users will just use either gpg or age.

Reply:
I would rename `changes` to `serviceRestarts`.

Reply:
Problem is that it also prints changes when nothing is restarted, for example `adding secret: mysecret`